server/src/main/java/org/apache/druid/server/initialization/jetty/CustomExceptionMapper.java

```diff
@@ -30,14 +32,26 @@
 @Provider
 public class CustomExceptionMapper implements ExceptionMapper<JsonMappingException>
 {
+  private final ServerConfig config;
+
+  @Inject
+  public CustomExceptionMapper(ServerConfig config)
+  {
+    this.config = config;
+  }
+
   @Override
   public Response toResponse(JsonMappingException exception)
   {
+    final ImmutableMap.Builder<Object, Object> builder = ImmutableMap.builder()
+        .put("error", "unable to process JSON input");
+
+    if (config.isShowDetailedJettyErrors()) {
```

Comment:
This would probably require a docs change no ?

Reply:
Updated, thanks